source/jormungandr/jormungandr/interfaces/v1/Journeys.py

```diff
@@ -382,6 +382,24 @@ def __init__(self):
             hidden=True,
             help="how long it takes to park the car, " "used especially in distributed scenario",
         )
+        parser_get.add_argument(
+            "_here_realtime_traffic",
```

Comment:
If think it would be better not to mentioned `here` in the input parameters as they could be used for other connectors in the near future. 
```suggestion
            "_realtime_traffic",
```
Same applies for the above new param.

Reply:
I did it on purpose to prefix with Here, in case of multi street network core, like Asgard for the bike and Here for the car. This is just to avoid collision between identical parameters.
I see the future :mage_man:

Reply:
I don't how this should be handled.... For the moment, we can only limite users' access to certain `coverage` by their token, but not the feature... I mean we cannot close Here for user A and open Here for user B on the same coverage... there is probably something to do **on the authentication level**...  For me, and this parameter should be defaulted to `true`, we can trun off this feature if user's token doesn't allow the usage...